### DISCUSSION TITLE
Can we integrate a React MFE with Angular JS project(parent project) using webpack(or webpack module federation) ?

### DISCUSSION THREAD

**Discussion by git-overflow** [webpack/webpack#13696] [Q&A]
We have a parent or host project which is on Angular JS and we are creating a new React app, and would like to integrate React app as MFE with the host project. .

The Angularjs project has Grunt and other java based services and we do not want to change lot of configurations in host project.
Wanted to know which of these options will be feasible to load the react app at run time:

1.  Load the react app in Angularjs using system js module and webpack configuration or
2. Can I convert react app as single spa app and load it in Angularjs project without converting the Angularjs project to single spa? I can use import map for loading React app in Angularjs project.(Is it possible to load a MFE single-spa child app with parent app being as-is )
3. Can we use webpack module federation in the Angular JS project to do the MFE integration

Thanks in advance for any examples or suggestions that you provide

**Comment by alexander-akait** [comment]
Better ask this in stackoverflow, but yes you can

**Reply by git-overflow** [reply]
Thanks for the suggestion. If module federation isn't possible, what are the other alternatives?

With single-spa MFE integration approach, it is required to convert the current host project as single-spa MFE Angular JS which requires us to do lot of changes in  the parent project and therefore, looking to go via webpack way for MFE integration
Is it possible to import the React MFE via webpack 4/3 versions in the host Angular JS, and do you have any inputs/examples for me to dig deeper into it

Thanks!
